themes/survey/ls6_surveytheme/views/layout_errors.twig

```diff
@@ -53,35 +52,38 @@
 
 <body class="{{ aSurveyInfo.class.body }} darkmode-{{aSurveyInfo.options.darkmode }} lang-{{ aSurveyInfo.languagecode }} {{ surveyformat }}" marginwidth="0"
       marginheight="0" {{ aSurveyInfo.attr.body }}>
-<div class="container">
-    <div id="error-card" class="card w-75 mt-6 mb-0 mx-auto">
-        <div class="card-body">
-            <h2 class="card-title">
-                {% if aError.error %}
-                    {{ aError.error }}
+    <div class="container m-auto ">
+        <div class="row align-items-center">
+            <div class="col">
+                <h2 class="error-title">
```

Comment:
error title especially and the page in general doesn't look exactly like design in figma